tests/e2e/workloads/bdi/test_bdi_workload.py

```diff
@@ -0,0 +1,54 @@
+from tests.e2e.workloads.bdi.bdi_base_class import TestBdiWorkloadBaseClass
+from ocs_ci.framework.pytest_customization.marks import (
+    skipif_ceph_capacity_less_equal_1T,
+)
+
+
+class TestBdiWorkloadSF1(TestBdiWorkloadBaseClass):
```

Comment:
better use one test class with parameterize with different markers so each test can be run as stand-alone

Reply:
I will change on the next PR